starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -15,4 +15,16 @@
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+<button (click)="addContact()">Submit New Contact</button>
+
+
+<div class="container-contact">
+  <div *ngFor="let contact of contacts">
+    <li> Name : {{contact.name}}</li>
+    <li>Email : {{contact.email}}</li>
+    <li>Phonenumber : {{contact.phoneNumber}}</li>
+    <li>
+      <img class="img" src="{{contact.image}}">
```

Comment:
That would work, but better get used to : ```<img [src]="contact.image">```.
Furthermore, it is best practice to always include descriptive alt text (if the image is not purely decorative), e.g.: ```alt="Profile picture of {{ contact.name }}"```.